src/arguments.cpp

```diff
@@ -254,6 +254,12 @@ Error Arguments::parse(const char* args) {
             CASE("log")
                 _log = value == NULL || value[0] == 0 ? NULL : value;
 
+            CASE("loglevel")
```

Comment:
I predict that after a couple of years this method will contain more than 500 "CASE" blocks.
Someone from the contributors really needs to do refactoring here using normal patterns (factory and so on ...)